src/containers/App/App.js

```diff
@@ -15,10 +15,11 @@ import WithDivaAuthorization from '../../util/WithDivaAuthorization';
 
 import SideMenu from '../../components/SideMenu/SideMenu';
 
-import Home from '../../containers/Home/Home';
-import MyAccount from '../../containers/MyAccount/MyAccount';
-import MyHome from '../../containers/MyHome/MyHome';
-import UserInfo from '../../containers/UserInfo/UserInfo';
+import  Home from '../../containers/Home/Home';
```

Comment:
Why the extra spaces ;)

Reply:
Yeah this happens without a linter... ;)